feature-group-definitions/array-at.yml

```diff
@@ -1,4 +1,5 @@
-name: Array at()
+name: Array at
```

Comment:
I've used the shortest possible text for `name` and got more detailed in `description`. I didn't see documented guidelines for any of this. Do you have pointers?

Reply:
We have no guidelines for when the name should look like code or not, but some precedent. So far we have these names with `()` in them:
- :has()
- :indeterminate()
- :is()
- :where()
- Array at()
- Array flat() and flatMap()
- checkVisibility()
- color-mix()
- CSS abs() and sign()
- CSS calc()
- CSS calc() constants e, pi, infinity, and NaN
- CSS min(), max(), and clamp()
- CSS pow(), sqrt(), hypot(), log(), exp()
- CSS round(), mod(), and rem()
- CSS sin(), cos(), tan(), asin(), acos(), atan(), and atan2()
- image-set()
- input.showPicker()
- scrollIntoView()
- select.showPicker()
- structuredClone()

Roughly, my thinking has been that when adding `()` or other small pieces of syntax hints towards what kind of thing it is, then do it even though it's not minimal. My thought has been that we'll eventually mark up names with `<code>` and have names like this:
- `:has()`
- Array `flat()` and `flatMap()`
- CSS `abs()` and `sign()`
- `input.showPicker()`
- `structuredClone()`

I think this has very little downside in the cases of global/static things like `:has()` or `structuredClone()`.

For instance methods (stuff on prototypes) it's a little bit inconsistent, already. I think we should favor short names, and for Array the style works.

For HTML elements it's a bit awkward and I'm not sure the choices so far will stand, but the alternatives to `input.showPicker()` that I've considered are worse IMHO:
- `HTMLInputElement.prototype.showPicker()` (much worse)
- `HTMLInputElement` `showPicker()`
- `<input>` `showPicker()`

@ddbeck to the rescue?

Reply:
I've accepted your suggestions. It would probably be good to split this discussion out to an issue so that we can document the guidelines for the `name` field.

Reply:
I've copied my comment to https://github.com/web-platform-dx/web-features/issues/548#issuecomment-1951652827.